<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/ensam/assets/imgs/ensam_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#backButton{
	border: 0;
	border-radius:10px;
	background:#20518A;
	color:white;
	font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="chartWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>941</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ganttButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>84</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Gantt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tfrButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>84</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TFR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tarButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>84</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TAR</string>
   </property>
  </widget>
  <widget class="QFrame" name="cdsFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="sequence_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt bold &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sequence :</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="ttFrame">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="tt_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt bold &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TT :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(0, 0, 0, 50);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Developed by &lt;a href=&quot;https://www.linkedin.com/in/karim-gehad/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;Karim Gehad&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
